src/borg/testsuite/archiver/extract_cmd.py

```diff
@@ -309,12 +309,12 @@ def test_extract_include_exclude_regex_from_file(self):
         self.create_regular_file("file3", size=1024 * 80)
         self.create_regular_file("file4", size=1024 * 80)
         self.create_regular_file("file333", size=1024 * 80)
-        self.create_regular_file("aa:something", size=1024 * 80)
+        self.create_regular_file("aa-something", size=1024 * 80)
 
         # Create while excluding using mixed pattern styles
         with open(self.exclude_file_path, "wb") as fd:
             fd.write(b"re:input/file4$\n")
-            fd.write(b"fm:*aa:*thing\n")
+            fd.write(b"fm:*aa-*thing\n")
```

Comment:
we already talked about this one on irc:

that aa:something filename was just used to test pattern prefix vs. filename confusion and that only can be tested with a 2-letter plus colon prefix. as the colon is invalid on windows, you just removed it. but that makes the whole test pointless, so i'ld suggest removing that file and the match on it.

instead, for testing a fm: pattern, it could also match `fm:file3*` and check if it matches the correct 2 files for that.